lib/deploy/migration.rb

```diff
@@ -0,0 +1,13 @@
+module Deploy
+  module Migration
```

Comment:
My one thought here is that it would be nice to give this a more descriptive name, perhaps something like `Deploy::MigrationDatabaseTimeout` so that it is more clear what's being done in the initializer where we are using it.